### PR TITLE
Clone random mutants of PB2

### PR DESCRIPTION
**Pull request by ericmjl**
@Starnite take it off from here! :)

### CODE REVIEW

**Comment by ericmjl** [comment]
@Starnite: the CPEC plate with 6 colonies is currently in the deli fridge, 2nd shelf from the top, left half, wrapped with parafilm. The other transformation plate should be ready tomorrow for growing up and then miniprepping on Friday.

**Comment by ericmjl** [comment]
@Starnite: updated with bacterial culture protocol in this branch. Not available yet in master; when done with mini prepping, and measuring DNA concentration, I think that's sufficient for this branch to be merged into master.

File: experiments/20161018-random-pb2-mutants/20161018-random-pb2-mutants.md [hunk 1]
```diff
@@ -184,3 +184,4 @@ About 20 hours later, Vivian miniprepped the bacteria culture. DNA concentration
 ## Transfection
 
 2016-10-31: Seeded 1:40 dilution (25ul cells in 1mL DMEM) of passage-3 293T cells in a 12-well plate. The 8 mutant strains with the highest DNA concentration will be assayed.
+- Cell count upon seeding: 74.5*40*10E4*(25/1000) = 7.4 million cells per well
```
Comment (ericmjl) [inline experiments/20161018-random-pb2-mutants/20161018-random-pb2-mutants.md]: @Starnite: watch out the formatting! `*` in markdown is intended for italicization. I'm going to directly update the note with a changed formatting, so don't forget to sync up.

**Review by ericmjl — COMMENTED** [review]
@Starnite: I think we can merge in this PR, the goals for the branch have been achieved.

**Review by Starnite — APPROVED** [review]

**Pull request merged**